src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,32 @@
+import {
+  Heading,
+  Notification as DSNotification,
+} from "@nypl/design-system-react-components"
+
+/**
+ * Custom RC Notification component that internally uses the DS `Notification`
+ * component. The header level and aria-label are intentionally hard-coded.
+ * For now, this notification should only display on the home and search result
+ * pages and that is controlled in the `Layout` component.
+ */
+const Notification = ({ notification = "" }) => {
+  if (!notification) return null
+
+  return (
+    <DSNotification
+      aria-label="Research Catalog Alert"
+      notificationHeading={
+        <Heading level="h2">New Service Announcement</Heading>
+      }
+      notificationContent={
+        <div dangerouslySetInnerHTML={{ __html: notification }} />
```

Comment:
why are you using `dangerouslySetInnerHtml` here?

Reply:
In case the notification is not a string and includes html. I guess we don't have notifications such as `Urgent: the library is <b>closed</b>`, so this could just be a string.

Reply:
We actually do use html in the notification env var! Specifically the <a> tag is an example that comes to mind. That said, I don't think the notification ends up being anything but a string, given that it's historically been input in the AWS console.